src/ClientProtos/capture_data.proto

```diff
@@ -175,4 +175,5 @@ message Color {
 message CaptureDeserializerFuzzerInfo {
   CaptureInfo capture_info = 1;
   repeated TimerInfo timers = 2;
+  string filename = 3;
```

Comment:
This is only used for the fuzz test? There should be a comment about that.

Reply:
Well the whole proto ist just for the fuzz test. And there is a comment explaining that

Reply:
I will go ahead and merge that. I hope that's okay. :innocent: